public/js/models.js

```diff
@@ -424,7 +424,9 @@ models.User = Backbone.Model.extend({
             superuser: false,
             displayName: "[unknown]",
             link: null,
-            emails: []
+            emails: [],
+            preferredContact: [],
+            noShare: "",
```

Comment:
Hmmm... with falses/nulls/undefineds above, this is empty string.  Let's pick one for a default state and use it consistently.